packages/react-composites/src/composites/CallWithChatComposite/CallWithChatComposite.tsx

```diff
@@ -172,13 +172,29 @@ const CallWithChatScreen = (props: CallWithChatScreenProps): JSX.Element => {
     setActivePane('none');
   }, [setActivePane]);
 
+  /** Constant setting of id for the parent stack of the composite */
+  const compositeParentDivId = 'callWithChatCompositeParentDiv-internal';
```

Comment:
nit: use `root` instead of `parent`. To Contoso the parent of callwithchat is whatever they have put it in